_data/simple-icons.json

```diff
@@ -1446,7 +1446,7 @@
         },
         {
             "title": "Aston Martin",
-            "hex": "000000",
+            "hex": "161A11",
```

Comment:
```suggestion
            "hex": "F4F4F2",
```
We can stick to the color they use in the logo itself. Although their brand color is `#00665e`, I was hard pressed to find any usage. They use the green background with white logo majority of the times.
![astonmartin](https://github.com/simple-icons/simple-icons/assets/35000807/b19e18b6-06f4-4b2e-9e1d-4abdca5da19e)

Reply:
@PratyushJayachandran The `hex` is there to store the brand color, not the logo color. In this case, I would really expect `#00665E`.

Reply:
According to this [page](https://www.astonmartinhouston.com/about-us/aston-martin-logo-history/) ` Aston Martin settled on its contemporary logo in the iconic silver, black, and green color scheme `

In their website the logo uses ` #fff ` on black bg and ` #161A11 ` on white bg, and as per our guidelines in the absence of brand guidelines we use the brand's primary web color which i believe this ` #161A11 ` so i think we should stick to it.

@PeterShaggyNoble What do you think?